### DISCUSSION TITLE
How To rotate dataLabel in angle 45 degree ?

### DISCUSSION THREAD

**Discussion by jiaulislam** [apexcharts/apexcharts.js#3882] [Q&A] [answered]
Hi everyone, I'm trying to rotate dataLabel in 45 degree. I couldn't find any documentation on that. Can anyone please tell how to rotate the dataLabel of bar chart (red marked on image).

Searched on : https://apexcharts.com/docs/options/datalabels/

TIA

![image](https://github.com/apexcharts/apexcharts.js/assets/54698236/54341587-f447-473c-8d59-0f126262ba48)

**Comment by INFIASHUTOSH** [comment]
did you find any solution? please share

**Reply by jiaulislam** [reply, marked as answer]
@INFIASHUTOSH use the `angle` property to rotate the label.

```js
<XAxis
          dataKey={"label"}
          angle={-30}
          textAnchor="end"
          height={70}
          tick={{
            stroke: TICK_COLOR,
            strokeOpacity: 0.1,
            fontSize: LABEL_TICK_FONT_SIZE,
          }}
        />
```

**Comment by amirdbt** [comment]
is the above for Apex charts?
Seems like its for Recharts, is there a way to achieve this in React Apex charts
